docs/components/modal-fullscreen.md

```diff
@@ -472,15 +472,19 @@ export default {
   --KModalFullscreenColor: green;
 }
 
-.ml-25 {
+.horizontal-spacing {
   margin-left: 25%;
 }
 
+.button-spacing {
+  margin-right: var(--kui-space-40, $kui-space-40) !important;
+}
+
 .k-switch {
-  border-top: 1px solid #eaecef;
-  padding-top: 26px;
-  border-bottom: 1px solid #eaecef;
-  padding-bottom: 26px;
+  border-top: var(--kui-border-width-10, $kui-border-width-10) solid #eaecef;
+  padding-top: var(--kui-space-80, $kui-space-80);
+  border-bottom: var(--kui-border-width-10, $kui-border-width-10) solid #eaecef;
+  padding-bottom: var(--kui-space-80, $kui-space-80);
```

Comment:
If this is in the markdown page docs styles section, you don't need the CSS variables and can just use the Sass tokens. 

They are only to be put in place where we need to allow for customization like in the component itself.